src/test/java/fi/hsl/transitdata/hslalertsource/HslAlertPollerTest.java

```diff
@@ -38,7 +39,7 @@ public void testFeedMessage(String filename, int expectedEntities, int expectedT
     private void validateInternalMessage(GtfsRealtime.TripUpdate update) {
         final GtfsRealtime.TripDescriptor trip = update.getTrip();
         final int joreDirection = trip.getDirectionId() + 1;
-        final InternalMessages.TripCancellation cancellation = HslAlertPoller.createPulsarPayload(trip, joreDirection, trip.getTripId(), InternalMessages.TripCancellation.Status.CANCELED);
+        final InternalMessages.TripCancellation cancellation = HslAlertPoller.createPulsarPayload(trip, joreDirection, trip.getTripId(), InternalMessages.TripCancellation.Status.CANCELED, Optional.empty());
```

Comment:
I think it would be good to have a unit test for both cases. Specially that the formatting of the datetimesring doesn't change.